### PR TITLE
OCPBUGS-36532: Agent installer wait-for, just test connectivity to host

### PR DESCRIPTION
**Pull request by bfournie**
In the agent wait-for, when the APIs are not available, an attempt is made to check the host by ssh'ing to it. This results in confusing debug messages if the keys aren't present. This changes the check to just test connectivity to the host.

### CODE REVIEW

File: pkg/agent/cluster.go [hunk 1]
```diff
@@ -483,14 +481,16 @@ func (czero *Cluster) PrintInstallStatus(cluster *models.Cluster) {
...
-// CanSSHToNodeZero Checks if ssh to NodeZero succeeds.
+// CanSSHToNodeZero Checks if NodeZero is reachable.
 func (czero *Cluster) CanSSHToNodeZero() bool {
 	ip := czero.API.Rest.NodeZeroIP
 	port := 22
 
-	_, err := ssh.NewClient("core", net.JoinHostPort(ip, strconv.Itoa(port)), czero.API.Rest.NodeSSHKey)
+	conn, err := net.Dial("tcp", fmt.Sprintf("%s:%d", ip, port))
 	if err != nil {
 		logrus.Debugf("Failed to connect to the Rendezvous Host: %s", err)
```
[the remark this answers is not in the document]
Comment (bfournie) [reply pkg/agent/cluster.go]: Changed function name and clarified error message to indicate connecting over port 22.

**Review by rwsu — COMMENTED** [review]
/lgtm

**Pull request merged**